timemachine/potentials/nonbonded.py

```diff
@@ -274,6 +276,9 @@ def nonbonded_v3_on_specific_pairs(conf, params, box, pairs, beta: float, cutoff
         can be returned.
     """
 
+    if len(pairs) == 0:
+        return 0.0, 0.0
```

Comment:
nit: makes return type unstable (array, array) or (float, float) -- would it be better to return zero arrays in this case instead?
```suggestion
        return np.zeros(1), np.zeros(1)
```
Or refactor call site not to call this function when `len(pairs) == 0`?

Reply:
good call on the typing